### ISSUE TITLE
Decide whether to make a single page application or associate the web app with the API

### ISSUE THREAD

**Issue by ryan-bradford** [ReGame-VR/WHAM-api#1]
The two/three options are:

1. Have a single-page web application that uses React or Angular or some other Web framework to render content. The benefits of this are that it would be very customizable with animations and JS code. The drawbacks are that having a separate whole application will create more maintenance work and more time to develop.

2. Keep going the same way we are and use Handlebars to serve content directly from the API. With this option, because the data will be tied directly to the rendering, errors will be less bound to happen. The drawback is that animations and fancy graphics might be harder.

3. Keep using a templating engine to render the content, but switch to a different engine that allows more customizability and JS code. I'm sure there are many options, but I don't know which one's are best.

**Comment by ryan-bradford** [comment]
As for my thoughts, I lean more towards option 2. We've already done quite a bit of work on the handlebars templates and keeping with them won't be too hard. Option 3 is also appealing, but I'd tend to want to switch only once we hit a road block. Option 1 scares me because I don't know if we have the human capital to maintain 2/3 applications (Mobile + Web) and the API.

**Comment by ryan-bradford** [comment]
In the end, after a discussion, we are deciding to leave the Web code as a dynamic and templated rather than a single page application. This will allow us to only have two repositories: API and Mobile. In the long run, this should help with less bugs and more features.
